app/src/main/res/values/strings.xml

```diff
@@ -549,30 +550,30 @@
     <string name="list_create_hint">name of new list</string>
     <string name="list_name_empty">please input name of new list.</string>
     <string name="list_created">new list created.</string>
-    <string name="list_delete_confirm">\"%1$s\" list will be deleted. Are you sure?</string>
+    <string name="list_delete_confirm">Delete the \"%1$s\" list?</string>
     <string name="list_member">List users</string>
     <string name="list_timeline">List timeline</string>
-    <string name="list_member_delete_confirm">\"%1$s\" will be deleted from \"%2$s\". Are you sure?</string>
+    <string name="list_member_delete_confirm">Delete \"%1$s\" from \"%2$s\"?</string>
     <string name="target_user">Target user</string>
     <string name="list_owner">List owner</string>
-    <string name="list_create">Create new list…</string>
+    <string name="list_create">New list…</string>
     <string name="list">List</string>
-    <string name="list_not_supported">this instance does not support list.</string>
-    <string name="cant_access_list">Can\'t access to lists</string>
-    <string name="account_sync_failed">account sync failed.</string>
-    <string name="list_retry_with_follow">You are not following %1$s yet. Would you like to follow and then add to list?</string>
-    <string name="cant_add_list_follow_requesting">Can\'t add user to list:\nPlease retry 2…3 times. Or if user is \"locked\", please wait until follow request is approved.</string>
-    <string name="cant_add_list_follow_failed">Can\'t add user to list:\nfollow operation failed.</string>
-    <string name="list_member_added">User has been added to the list.</string>
-    <string name="confirm_mute_notification_for_user">Mute notification from this user</string>
-    <string name="account_moved_to">%1$s has moved to:</string>
-    <string name="jump_moved_user">%1$s has been moved. Would you like to check %2$s ?</string>
+    <string name="list_not_supported">this instance does not support lists.</string>
+    <string name="cant_access_list">Can\'t access lists</string>
+    <string name="account_sync_failed">Could not sync account.</string>
+    <string name="list_retry_with_follow">Follow %1$s and add to list?</string>

```

Comment:
"follow then add to list" feature is bit tricky because follow processing may take a time on servier side and add-to-list may faiulre while follow processing is not complete. The sentences related to this function should be redundant